gen/template/lib/__app__/commands.rb

```diff
@@ -0,0 +1,22 @@
+require '__app__'
+
+module __App__
+  module Commands
+    # No point in using autoload/autocall here; it's loaded immediately by
+    # `register` calls.
+    Registry = CLI::Kit::CommandRegistry.new(
+      default: 'help',
+      contextual_resolver: nil
+    )
+
+    def self.register(const, cmd, path)
+      autoload(const, path)
+      Registry.add(->() { const_get(const) }, cmd)
+    end
+
+    register :Add,      'add',      '__app__/commands/add'
```

Comment:
Should we just get rid of these? Maybe just leave `help` and `example` command?

Reply:
done.